src/backend/aspen/api/views/samples.py

```diff
@@ -194,20 +194,20 @@ async def update_samples(
         for key, value in update_data:
             if key in ["collection_location", "sequencing_date"]:
                 continue
-            if value is not None:  # We need to be able to set private to False!
-                setattr(sample, key, value)
+            setattr(sample, key, value)
         # Location id is handled specially
         if update_data.collection_location:
             loc = await db.get(Location, update_data.collection_location)
             if not loc:
                 raise ex.BadRequestException("location is invalid")
             sample.collection_location = loc
+
         # Sequencing date is handled specially
-        if update_data.sequencing_date:
-            sample.uploaded_pathogen_genome.sequencing_date = (
-                update_data.sequencing_date
-            )
+        sample.uploaded_pathogen_genome.sequencing_date = update_data.sequencing_date  # type: ignore
```

Comment:
What happens in the case that a user does not want to make a change to the sequencing date? Will the old sequencing date be in the request? Sequencing date is nullable, so I am worried that any existing date could be accidentally overwritten.

Reply:
so how this endpoint is being used currently is that all field data is being sent in, so it will send either the  old sequencing date (if the user does not want to make changes), the new sequencing date, or null if they wish to delete the value. so to answer your question yes the old sequencing_date will be sent in the request.